doc/content/source/materials/ConcreteThermalMoisture.md

```diff
@@ -376,73 +396,101 @@ mechanisms separately. Instead, they tend to reproduce the general combined tren
 
-It's obvious that all three reference moisture diffusivity models strongly depend on the value of humidity $H$, and indirectly on the temperature $T$. Once the value of
-reference moisture diffusivity $D_{h,0}$ is obtained, the actual concrete moisture diffusivity $D_h$ required by the moisture diffusion governing equation, [!eqref](moisture_governing),
-can then be calculated by
+It has been reported by [!cite](bazant1982finite) that the additional moisture diffusion due to thermal gradients included in the moisture governing equation is negligible. Thus the value of $D_{ht}$ is set to $1.0\times10^{-5} \times D_h$ by default. This parameter can, however, be set to an arbitrary value if desired.
 
-!equation id=Dh_coupled
-D_h = \left\{\begin{array}{lll}
-        D_{h, 0}f_1(T) & \text{for} & T \le 95 \degree C \\
-        D_{h, 0}f_1(95\degree C)f_2(T) & \text{for} & T > 95 \degree C
-    \end{array}\right.
+Since the values of parameters such as $C_{a}$ and $C_{w}$ are not provided in the numerical model, they are taken from different references. [!cite](poyet2009temperature) observed that $C_{a}$ vary between 42 kJ/mol and 80 kJ/mol for concrete, therefore, an average value is considered for general concrete, i.e., 60 kJ/mol. $C_{w}$ is obtained as follows
+
+!equation id=Cw
+C_w = \rho_{water} c_p
+
+where \rho_{water} is density of water (1000 kg/m$^3$) and $c_p$ is heat capacity of water (= 4180 J/kg-$\degree C$)
+
+
+3. [!cite](mensi1988sechage)
+
+!equation id=Dh0_mensi
+D_{h} = Ae^{BC}
 
 where
 
-!equation id=f1T
-\begin{array}{lll}
-    f_1(T) = \exp\left(\frac{Q}{R}\left(\frac{1}{T_{ref}}-\frac{1}{T}\right)\right) & \text{for} & T \le 95° C
-\end{array}
+$A = 1.042 \times 10^{-13}  m^2/s$ \\
+$B = 0.05$ \\
+$C$ = initial water content in concrete (kg/m$^3$)
 
-in which $T$ is the absolute temperature ($K$), $Q$ is activation energy for water migration along the adsorption layers in the necks, and $R$ is gas constant
-with $Q/R$=2700 K, and
+$C$ is given by
 
-!equation id=f2T
-\begin{array}{lll}
-    f_2(T) = \exp\left(\frac{T-95}{0.881 + 0.214(T-95)}\right) & \text{for} & T > 95 \degree C
-\end{array}
+!equation id=C_mensi
+C = w/c \times M_{cement}
 
-### Coupled moisture diffusion by thermal gradient, $D_{ht}$
+where $M_{cement}$ mass of cement in concrete mix.
 
-It has been reported by Bazant et al. [!cite](bazant1982finite) that the additional moisture diffusion due to thermal gradients included in the moisture governing equation is negligible. Thus the value of $D_{ht}$ is set to $1.0\times10^{-5} \times D_h$ by default. This parameter can, however, be set to an arbitrary value if desired.
+Since no coupling between heat and moisture transfer is considered the value of $D_{ht}$, $C_{a}$, are $C_{w}$ considered as zero.
```

Comment:
No coupling between heat and moisture transfer is considered in this model, so the values of ... are set to zero.